src/histogram.rs

```diff
@@ -246,8 +247,14 @@ impl Histogram {
 }
 
 impl Histogram {
-    /// `observe` adds a single observation to the `Histogram`.
-    pub fn observe(&self, v: f64) {
+    /// `observe_seconds` adds a single observation (with seconds) to the `Histogram`.
+    pub fn observe_seconds(&self, v: f64) {
+        self.core.write().unwrap().observe(v)
+    }
+
+    /// `observe` adds a single observation (with Duration) to the `Histogram`.
+    pub fn observe(&self, d: Duration) {
```

Comment:
Observations can be thinks other than durations, the original interface is correct.